src/controller/credential_controller.ts

```diff
@@ -89,8 +89,8 @@ export async function issueVC(
       authorization,
       registry: registryParsed.identifier,
       signCallback: async ({ data }) => ({
-        signature: issuerKeys.assertionMethod.sign(data),
-        keyType: issuerKeys.assertionMethod.type,
+        signature: issuerKeys.authentication.sign(data),
```

Comment:
Did'nt get why you changed the signature

Reply:
we have used `keyUri` as the one from 'authentication' section, but used signature from assertionMethod. Which fails the validation of the record.

Reply:
ok got it